src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressionManager.cs

```diff
@@ -18,6 +21,8 @@ public class SuppressionManager
 
 		private static SuppressionManager Instance { get; set; }
 
+		private static readonly Regex _suppressPattern = new Regex(@"Acuminator\s+disable\s+once\s+(\w+)\s+(\w+)");
```

Comment:
Good idea.
```suggestion
		private static readonly Regex _suppressPattern = new Regex(@"Acuminator\s+disable\s+once\s+(\w+)\s+(\w+)", RegexOptions.Singleline | RegexOptions.Compiled);
```

Reply:
According to MSDN (https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regexoptions?view=netframework-4.8#System_Text_RegularExpressions_RegexOptions_Singleline):
> Specifies single-line mode. Changes the meaning of the dot (.) so it matches every character (instead of every character except \n).

Since your pattern doesn't contain any (.) characters the SingleLIne option is redundant:
```suggestion
		private static readonly Regex _suppressPattern = new Regex(@"Acuminator\s+disable\s+once\s+(\w+)\s+(\w+)", RegexOptions.Compiled);
```